install/dataloggers.csv

```diff
@@ -406,6 +406,10 @@ Kinemetrics,BASALT 8X DATALOGGER,1626,Richmond Hill Borehole,,2013-01-28T00:00:0
 Kinemetrics,BASALT 8X DATALOGGER,1626,Redcliffs Borehole,,2013-09-03T00:00:01Z,9999-01-01T00:00:00Z
 Kinemetrics,BASALT 8X DATALOGGER,1627,Redcliffs Borehole,,2013-01-28T00:00:01Z,2013-09-03T00:00:00Z
 Kinemetrics,BASALT 8X DATALOGGER,1627,Richmond Hill Borehole,,2014-07-22T03:00:01Z,2014-10-30T00:30:00Z
+Kinemetrics,FBA-ES-T-BASALT,1196,Summerhill,,2013-04-18T00:10:00Z,2014-10-14T00:00:00Z
+Kinemetrics,FBA-ES-T-BASALT,1275,Summerhill,,2014-10-14T00:00:05Z,9999-01-01T00:00:00Z
+Kinemetrics,FBA-ES-T-BASALT,1677,Summerhill,,2013-01-16T00:30:00Z,2013-04-18T00:00:00Z
+Kinemetrics,FBA-ES-T-BASALT,556,Summerhill,,2012-10-16T04:30:00Z,2013-01-15T11:00:00Z
 Kinemetrics,Obsidian 4X Datalogger,139,Deep Cove,Strong Motion,2015-10-22T05:00:01Z,2020-06-23T00:48:00Z
```

Comment:
This is a bit tricky. From what I understand the "datalogger" is actually a strong motion recorder `Kinemetrics | BASALT` acting as a datalogger whereby we are using the spare 4th channel for geomag (the first 3 channels are for strong motion data).

So we probably need to add `Kinemetrics | BASALT` as a datalogger asset in delta that also matches the datalogger install entries above. But these assets will already exist in recorders (as that's what they are and are also being used for) so I'm not sure if this will create a Travis CI nightmare with the same serial numbers used across multiple asset types...

cc @ozym

Reply:
Keep the assets as is, install recorder, and extra sensor. The add a connection, and an offset to the 4th channel.

It will probably break but that would be the starting point to avoid duplicate kit.